appleseed-Installer/Product.wxs

```diff
@@ -0,0 +1,632 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+	<Product Id="{EEC9E532-B45A-4B0A-B2AA-1762CFB002D8}" Name="appleseed-MAX" Language="1033" Version="1.0.0.1" Manufacturer="appleseed" UpgradeCode="{46137F8F-2BD6-499C-ADDA-EC812E8E978B}">
+		<Package InstallerVersion="300" Compressed="yes" Platform="x64" InstallScope="perMachine" />
+    <Icon Id="icon.ico" SourceFile="Resources\icon.ico"/>
+    <Property Id="ARPPRODUCTICON" Value="icon.ico" />
+
+
+    <UIRef Id="SetupDialogUI" />
+
+    <Property Id="MAX2017" Value="{}" Secure="yes" />
```

Comment:
We actually dropped support for 3ds Max 2017, so it can be removed from here too.